Build broken on master

Related to Docker-in-Docker code.

```
Step 4/14 : RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz          | tar -xvzf - --transform='s,[^/]*/,,g' -C /usr/local/bin/          && chmod u+x /usr/local/bin/docker
 ---> Running in caa123e76e95
[91m  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[0m[91m
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[0m[91m
100   454  100   454    0     0    852      0 --:--:-- --:--:-- --:--:--   851
[0m[91m
gzip: stdin: not in gzip format
[0m[91mtar: Child returned status 1
tar: Error is not recoverable: exiting now
[0mThe command '/bin/sh -c curl https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz          | tar -xvzf - --transform='s,[^/]*/,,g' -C /usr/local/bin/          && chmod u+x /usr/local/bin/docker' returned a non-zero code: 2
make[2]: *** [build] Error 2
make[2]: Leaving directory `/mnt/ephemeral/workspace/toil-rnaseq/docker/1.12.3.docker'
/bin/sh: 1: cd: can't cd to 1.11.2.docker
/bin/sh: 1: cd: can't cd to 1.10.3.docker
make[1]: *** [runtime-container.DONE] Error 2
make[1]: Leaving directory `/mnt/ephemeral/workspace/toil-rnaseq/docker'
make: *** [docker] Error 2
```


The URL being called: https://get.docker.com/builds/Linux/x86_64/docker-1.12.3.tgz now returns the following response:

```
<html>
<head><title>403 Forbidden</title></head>
<body>
<h1>403 Forbidden</h1>
<ul>
<li>Code: AccessDenied</li>
<li>Message: Access Denied</li>
<li>RequestId: 8167A22F37FFA17B</li>
<li>HostId: 74xHcHzKQBVyzd8RqgpnJbPwwreQV7rOO4MhE20SNAMhV1DfQnyUVBv9ihCanFTth4G8hYsUjok=</li>
</ul>
<h3>An Error Occurred While Attempting to Retrieve a Custom Error Document</h3>
<ul>
<li>Code: AccessDenied</li>
<li>Message: Access Denied</li>
</ul>
<hr/>
</body>
</html>
```

--- a/Makefile
+++ b/Makefile
@@ -63,6 +63,7 @@ sdist: check_venv
 	$(python) setup.py sdist
 clean_sdist:
 	- rm -rf dist
+	- rm -rf .pytest_cache
 
 
 test: check_venv check_build_reqs

--- a/docker/Dockerfile.template
+++ b/docker/Dockerfile.template
@@ -13,8 +13,8 @@ RUN apt-get update && apt-get install -y \
     ca-certificates
 
 # Get the Docker binary
-RUN curl https://get.docker.com/builds/Linux/x86_64/docker-DOCKERVER.tgz \
-         | tar -xvzf - --transform='s,[^/]*/,,g' -C /usr/local/bin/ \
+RUN curl https://download.docker.com/linux/static/stable/x86_64/docker-DOCKERVER-ce.tgz \
+         | tar -xvzf - -C /tmp && mv /tmp/docker/* /usr/local/bin/ \
          && chmod u+x /usr/local/bin/docker
 
 # Set up a virtual environment with the system site package option so Toil

--- a/docker/Makefile
+++ b/docker/Makefile
@@ -1,5 +1,5 @@
 # Definitions
-docker_versions = 1.12.3 1.11.2 1.10.3 #1.9.1 1.8.3 1.7.1 1.6.2
+docker_versions = 17.03.0 17.06.0 17.09.0 18.03.0 18.06.0
 gen_directories = $(docker_versions:=.docker)
 build_tool = runtime-container.DONE
 

